integration_test/playertesting/PlayerTesting.ts

```diff
@@ -36,6 +78,28 @@ export const expectEvent = async <T extends Event>(
   );
 };
 
+/**
+ * Calls the given function with the player instance and expects the given events to occur.
+ * @param fn The function to call.
```

Comment:
Same as with the single event expectation

Reply:
fixed in a4169682